### PR TITLE
ref(TS) Convert some features to TS

### PR DESCRIPTION
**Pull request by robertpin**

### CODE REVIEW

File: react/features/follow-me/middleware.ts [hunk 1]
```diff
@@ -124,7 +119,7 @@ function _onFollowMeCommand(attributes = {}, id, store) {
         const { conference } = state['features/base/conference'];
 
         // As this participant is not stored in redux store we do the checks on the JitsiParticipant from lib-jitsi-meet
-        const participant = conference.getParticipantById(id);
+        const participant = conference?.getParticipantById(id);
```
Comment (horymury) [inline react/features/follow-me/middleware.ts:122]: we don't have a type `Participant`? Would it be complicated to create it?

File: react/features/video-quality/middleware.ts [hunk 2]
```diff
@@ -24,7 +22,7 @@ MiddlewareRegistry.register(({ dispatch, getState }) => next => action => {
             const { resolution } = getState()['features/base/config'];
 
             if (typeof resolution !== 'undefined') {
-                dispatch(setPreferredVideoQuality(Number.parseInt(resolution, 10)));
+                dispatch(setPreferredVideoQuality(Number.parseInt(`${resolution}`, 10)));
```
Comment (horymury) [inline react/features/video-quality/middleware.ts:25]: why did we need the string templating?

**Review by horymury — CHANGES_REQUESTED** [review]

File: react/features/follow-me/middleware.ts (hunk 1 above)
Comment (robertpin) [reply react/features/follow-me/middleware.ts:122]: We do, but this isn't `Participant` it's `JitsiParticipant`. And `JitsiParticipant` should be a lib-jitsi-meet interface

File: react/features/video-quality/middleware.ts (hunk 2 above)
Comment (robertpin) [reply react/features/video-quality/middleware.ts:25]: `resolution` can be either number or string, but `parseInt` requires the first param to be string

**Review by horymury — APPROVED** [review]

**Pull request merged**